docs/antrea-network-policy.md

```diff
@@ -902,25 +904,39 @@ an `appliedTo` must resolve to. More information on ClusterGroups can be found i
 **serviceAccount**: This selects all the Pods which have been assigned a specific ServiceAccount.
 For more information on its usage, refer to [this section](#serviceaccount-based-selection).
 
-**ipBlock**: This selects particular IP CIDR ranges to allow as `ingress`
-"sources" or `egress` "destinations". These should be cluster-external IPs,
-since Pod IPs are ephemeral and unpredictable.
+**ipBlock**: This selects particular IP CIDR ranges to allow as `ingress` "sources"
+or `egress` "destinations". These should be cluster-external IPs, since Pod IPs are
+ephemeral and unpredictable.
+
+Starting from Antrea v2.2, Antrea-native policies now support and `except` field under
+ipBlock, which can be used to carve out specific CIDRs in the base IP range:
+
+```yaml
+ipBlock:
+  cidr: 10.168.1.0/16
+  except: 
+    - 10.168.128.0/24
+```
 
-**fqdn**: This selector is applicable only to the `to` section in an `egress` block. It is used to
-select Fully Qualified Domain Names (FQDNs), specified either by exact name or wildcard
+Note that at the time of policy creation, the CIDRs in the `except` list will be validated
+to be strictly within the base CIDR range. Policy creation will fail otherwise. In addition,
```

Comment:
I feel we can remove "In addition", as this sentence is not an addition of the first one.